src/app/home/home.component.html

```diff
@@ -3,7 +3,7 @@
   <button mat-icon-button (click)="toggleNav()" i18n-title title="Menu"><mat-icon>menu</mat-icon></button>
   <img src="assets/cropped-ole-ico-logo-32x32.png" class="ole-logo"/>
   <h1 i18n>Planet Learning</h1>
-  <span class="toolbar-fill"></span>
+  <span class="toolbar-fill warn-text-color warn-text-align" *ngIf="userservice.roles.length === 0 && !userservice.isUserAdmin">You are not authorized. Please contact administrator.</span>

```

Comment:
The `<span class="toolbar-fill"></span>` is a special element which is very important for the layout of our toolbars making the elements before it aligned to the left, and the elements after aligned to the right.

Modifying it/removing it (the *ngIf will remove it when the user has access) will break the layout:
![image](https://user-images.githubusercontent.com/9203229/43241138-cedcdc22-9067-11e8-8332-cba62fde2f42.png)

Please add the message as part of a different element.